<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>task2window</class>
 <widget class="QDialog" name="task2window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>435</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>749</width>
    <height>435</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>749</width>
    <height>435</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Решение ДУ второго порядка</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QLabel" name="lFunc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>281</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria Math</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>sys_equ.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="bEuler">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Рассчитать по
методу Эйлера</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bReturn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Вернуться</string>
   </property>
  </widget>
  <widget class="QLabel" name="lY">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria Math</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>x =</string>
   </property>
  </widget>
  <widget class="QLabel" name="lH">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria Math</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>h =</string>
   </property>
  </widget>
  <widget class="QLabel" name="lA">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria Math</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>a =</string>
   </property>
  </widget>
  <widget class="QLabel" name="lY_der">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria Math</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>y  =</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ds_b">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="minimum">
    <double>-99.000000000000000</double>
   </property>
   <property name="value">
    <double>0.300000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ds_y">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="minimum">
    <double>-99.989999999999995</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="lB">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria Math</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>b =</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ds_a">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="minimum">
    <double>-99.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ds_h">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="value">
    <double>0.003000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ds_x">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="value">
    <double>2.000000000000000</double>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableW">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>381</width>
     <height>401</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::CustomDashLine</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>80</number>
   </attribute>
   <column>
    <property name="text">
     <string>t</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>X</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Y</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Z</string>
    </property>
   </column>
  </widget>
  <widget class="QDoubleSpinBox" name="ds_z">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="minimum">
    <double>-99.000000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="lA_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria Math</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>z =</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
